sql/tidb-specific.md

```diff
@@ -282,6 +282,13 @@ set @@global.tidb_distsql_scan_concurrency = 10
 - This variable is used to set the priority of executing the `ADD INDEX` operation in the `re-organize` phase.
 - You can set the value of this variable to `PRIORITY_LOW`, `PRIORITY_NORMAL` or `PRIORITY_HIGH`.
 
+### tidb_force_priority
+
+- Scope: SESSION
+- Default value: `NO_PRIORITY`
+- This variable is used to change the default priority for statements executed on a TiDB server.  A use case is to ensure that a particular user that is performing OLAP queries receives lower priority than users performing OLTP queries.
+- You can set the value of this variable to `PRIORITY_LOW`, `NO_PRIORITY` or `PRIORITY_HIGH`.
```

Comment:
It's better to use
```
`NO_PRIORITY`, `LOW_PRIORITY`, `HIGH_PRIORITY` or `DELAYED`
```
tidb_force_priority uses SQL priority options as its available values which is not the same as tidb_ddl_reorg_priority does.